### PR TITLE
Fixed engineering lockers grammar

### PR DESCRIPTION
**Pull request by DamianX**
and the relative paths
and some useless returns
that I noticed when opening #15185

### CODE REVIEW

**Comment by ghost** [comment]
#15185 merged, now fix this one

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/structure/closet/secure_closet/engineering_chief
-	name = "Chief Engineer's Locker"
+	name = "chief engineer's locker"
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Maybe instead add an \improper as it's a job title?

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 2]
```diff
@@ -80,22 +72,18 @@
...
+	new /obj/item/clothing/head/welding(src)
+	new /obj/item/clothing/head/welding(src)
+	new /obj/item/clothing/head/welding(src)
+	new /obj/item/weapon/weldingtool/largetank(src)
+	new /obj/item/weapon/weldingtool/largetank(src)
+	new /obj/item/weapon/weldingtool/largetank(src)
 
 /obj/structure/closet/secure_closet/engineering_personal
-	name = "Engineer's Locker"
+	name = "engineer's locker"
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: same for here

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 3]
```diff
@@ -104,34 +92,32 @@
...
+	else
+		new /obj/item/clothing/glasses/scanner/meson(src)
+	new /obj/item/taperoll/engineering(src)
+	new /obj/item/taperoll/engineering(src)
+	new /obj/item/device/gps/engineering(src)
+	new /obj/item/clothing/glasses/scanner/material(src)
 
 /obj/structure/closet/secure_closet/engineering_atmos
-	name = "Atmospheric Technician's Locker"
+	name = "atmospheric technician's locker"
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: and here

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 4]
```diff
@@ -140,35 +126,34 @@
...
+	new /obj/item/weapon/gun/projectile/flare(src) //yay for emergency lighting
+	new /obj/item/ammo_storage/box/flare(src)
+	new /obj/item/device/rcd/rpd(src)
+	new /obj/item/device/analyzer(src)
+	new /obj/item/clothing/glasses/scanner/material(src)
+	new /obj/item/device/gps/engineering(src)
 
 /obj/structure/closet/secure_closet/engineering_mechanic
-	name = "Mechanic's Locker"
+	name = "mechanic's locker"
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: you get the gist

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm (hunk 1 above)
Comment (DamianX) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: yeah I was on the fence, that's more reasonable

**Comment by ghost** [comment]
Now, if we can get a picture of it working, all good

**Comment by DamianX** [comment]
image

**Pull request merged**